packages/sdk-v2/src/sdk/savings/react.ts

```diff
@@ -131,6 +131,10 @@ export const useSavingsStats = (requiredChainId: number, refresh: QueryParams["r
     apy: undefined
   };
 
+  const [, totalStaked, totalRewardsPaid] = results[0]?.value ?? []; // eslint-disable-line no-unsafe-optional-chaining
+  const staked = useG$Amount(totalStaked);
```

Comment:
you need to verify that EVERYWHERE you use useG$Amount you also pass the requiredChainId from the surrounding hook if relevant